src/session/chat_list.rs

```diff
@@ -108,7 +108,6 @@ impl ChatList {
     pub fn fetch(&self, client_id: i32) {
         RUNTIME.spawn(async move {
             functions::GetChats::new()
-                .offset_order(i64::MAX)
```

Comment:
Is `i64::MAX` set by default when omitted behind the lines? At least, I couldn't find anything in https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1get_chats.html.

Reply:
`offset_order` was removed in recent tdlib versions. The official docs are outdated, so I suggest compiling the docs from tdgrand directly using `cargo doc` and using that instead.